### PR TITLE
fix: Share link in success message 

### LINKED ISSUE #8812: Share link in success message
i clicked copy link in the congrats message when buying nft 

https://canary.kodadot.xyz/ahp/collection/63?sort=price_ASC&listed=true

- it copied this link, it should probably be a link to the nft no? 

https://canary.kodadot.xyz/ahp/gallery/63-406

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #8812

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI;

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

File: components/common/ShareDropdown.vue [hunk 1]
```diff
@@ -69,11 +70,14 @@ const { shareOnX } = useSocialShare()
 const isModalActive = ref(false)
 const sharingTxt = $i18n.t('sharing.nft')
 const realworldFullPathShare = ref(`${window.location.origin}${route.fullPath}`)
+const shareLink = computed(() =>
+  props.link ? props.link : realworldFullPathShare.value,
```
Comment (daiagi) [inline components/common/ShareDropdown.vue:74]:
```suggestion
props.link ??​ realworldFullPathShare.value

```

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 8.57 USD/DOT ~ 2.334 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0xfb0a36eb0abf1ee4194322ed9394a596a09646e2c5ed5ecabec13a650d0375f8](https://polkadot.subscan.io/extrinsic/0xfb0a36eb0abf1ee4194322ed9394a596a09646e2c5ed5ecabec13a650d0375f8)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**